common/filesystem.hpp

```diff
@@ -10,9 +10,13 @@
 #include <string>
 
 namespace tec {
+#if defined(__unix__)
+const std::string PATH_SEPARATOR = std::string("\\"); /// OS File system path separator
+#else
+const std::string PATH_SEPARATOR = std::string("/"); /// OS File system path separator
+#endif
```

Comment:
copy paste or no, it still looks weird.

Reply:
I do not have a non-windows system to verify it all still works. I can swap it, but I can't really test it.